docs/modules/getting-started/pages/wan.adoc

```diff
@@ -0,0 +1,282 @@
+
+The member is up and running now.
+
+[source,shell,subs="attributes+"]
+----
+2021-11-23 11:08:15,055 [ INFO] [main] [c.h.i.c.ClusterService]: [172.18.0.3]:5701 [london] [{full-version}] 
+
+Members {size:1, ver:1} [
+	Member [172.18.0.3]:5701 - bed20746-1505-449b-9f4a-548bcdbe12b8 this
+]
+----
+
+Note the member's IP address and port since you are going to use it while configuring WAN
+replication in the active cluster and connecting Management Center.
+
+== Step 2. Start the Active Cluster
+
+Now you'll start another Enterprise cluster called `tokyo` with a single member. This will be the **active** cluster
+which means it does not need WAN Replication to be configured.
+
+Since the configuration for WAN Replication is a bit more detailed, you are going to use a Hazelcast configuration
+file and pass it to the Docker command, instead of giving all the configuration options in a single command.
+
+. In an existing or new directory, place a hazelcast.yaml (or xml) file with the following content.
++
+[tabs]
+====
+YAML::
++
+--
+
+[source,yaml]
+----
+hazelcast:
+  cluster-name: tokyo
+  license-key: <your license key>
+  wan-replication:
+    london-wan-rep:
+      batch-publisher:
+        londonPublisherId:
+          cluster-name: london
+          target-endpoints: 172.18.0.3:5701
+  map:
+    cities: <1>
+      wan-replication-ref:
+        london-wan-rep: <2>
+          republishing-enabled: false
+----
+--
+<1> Name of the map you are going to create.
+<2> Name of the WAN Replication configuration (given under the `wan-replication` tag) which provides the details on where the map data will be replicated.
+
+XML::
++
+[source,xml]
+----
+<hazelcast>
+    <cluster-name>tokyo</cluster-name>
+    <license-key>"your license key"</license-key>
+    <wan-replication name="london-wan-rep">
+        <batch-publisher>
+            <cluster-name>london</cluster-name>
+            <publisher-id>londonPublisherId</publisher-id>
+            <target-endpoints>172.18.0.3:5701</target-endpoints>
+    <map name="cities">
+        <wan-replication-ref name="london-wan-rep">
+    </map>
+</hazelcast>
+----
+====
+. Open a new terminal and start the active cluster.
++
+[source,shell,subs="attributes+"]
+----
+docker run \
+     --network hazelcast-network \
+     -e JAVA_OPTS="-Dhazelcast.config=/opt/hazelcast/config_ext/hazelcast.yaml" -v ~/config-tokyo:/opt/hazelcast/config_ext hazelcast/hazelcast-enterprise:{full-version}
+----
+
+Here is your other single-member cluster:
+
+[source,shell,subs="attributes+"]
+----
+2021-11-23 11:39:14,198 [ INFO] [main] [c.h.i.c.ClusterService]: [172.18.0.4]:5701 [tokyo] [{full-version}] 
+
+Members {size:1, ver:1} [
+	Member [172.18.0.4]:5701 - 98d9a815-5eb3-4341-bec1-e9816cee44b5 this
+]
+----
+
+Note the member's IP address and port since you are going to use it while connecting Management Center.
+
+See xref:wan:defining-wan-replication.adoc#wanbatchreplication-implementation[Configuring WAN Replication] for
```

Comment:
I don't think users need this right now. Maybe move it to the end of the tutorial as next steps.